hashtag-actor/components/pubsub.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
the components dir can be added to gitignore as these files are generated by cli, in test